docs/index-test.md

```diff
@@ -120,7 +120,7 @@ end
 
 ### Small image
 
-![](https://assets-cdn.github.com/images/icons/emoji/octocat.png)
+![](../assets/images/search.svg)
```

Comment:
```suggestion
> ![](../../assets/images/search.svg)
```

Reply:
@JeffGuKang sorry for not merging this PR when it was first submitted!

I think the URL needs the suggested adjustment. And adding the block quote improves the layout, I think.